src/backend/web/handlers/admin/blueprint.py

```diff
@@ -0,0 +1,37 @@
+from flask import abort, Blueprint
+from google.appengine.api import users as gae_login
+
+from backend.web.handlers.admin.authkeys import authkeys_get, authkeys_post
+from backend.web.handlers.admin.event import event_detail, event_list
+from backend.web.profiled_render import render_template
+
+
+"""
+This is a special interface available to TBA admins to manage data
+"""
+
+admin_routes = Blueprint("admin", __name__, url_prefix="/admin")
+
+
+@admin_routes.before_request
+def require_gae_admin() -> None:
+    """
+    Ensure that only admins can access this blueprint
+    TODO: we should probably change this to use firebase auth?
+    """
+    print(f"user: {gae_login.get_current_user()}")
+    if not gae_login.is_current_user_admin():
```

Comment:
@ZachOrr are we okay continuing to depend on this, or would we rather use firebase users and roles or something

Reply:
ah, yeah, great catch